### PR TITLE
Readme alias dev interoperabilidad

### PR DESCRIPTION
**Pull request by luelher**
Adding information in README to config alias hostname and login correctly to Clave Unica in development environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ Those should be set as environment variables:
     $ export AWS_REGION=<aws-region> AWS_ACCESS_KEY_ID=<aws-key-id> AWS_SECRET_ACCESS_KEY=<aws-secret> S3_CODEGEN_BUCKET=<bucket-name>
     $ export SIGNER_API_TOKEN_KEY=<our-signer-key> SIGNER_API_SECRET=<our-signer-secret>
 
-After those variables are set, you just need to run:
+2. Set hostname alias: Must be set the alias `dev.interoperabilidad.digital.gob.cl` in the `/etc/hosts` file with the IP of docker machine. By default
```
Comment (leosoto) [inline README.md]: Must be set the alias -> You must set the alias...

File: README.md [hunk 2]
```diff
@@ -23,7 +23,13 @@ Those should be set as environment variables:
...
+this alias is config in `OP_CALLBACK_URL` to make correctly the login with Clave Unica.
+
+    $ docker-machine ls # To get the IP address of docker machine
```
Comment (leosoto) [inline README.md]: Whoa, are you using docker-machine? I think we used it when we started the project a while ago, but at least on Mac it's not longer needed. And AFAIU neither it is on Linux. So usually the `/etc/hosts` record should point to 127.0.0.1 unless something weird is going on (such as using docker-machine)

Comment (leosoto) [reply README.md]: Sounds good to me. Just a slight English fix:

[...] By default this alias is _configured_ via OP_CALLBACK_URL _to login correctly_ with Clave Unica.

Comment (afiebig) [reply README.md]: 👍

**Comment by luelher** [comment]
@leosoto I did the changes that you suggested, please take a look... thanks

**Pull request merged**